CHANGELOG.md

```diff
@@ -1,374 +1,506 @@
-================================
```

Comment:
this file should be rst file file not md file.

Reply:
Just follow the requirement from Anupam. Will be any issue if change to md file?

Reply:
Earlier we used to have the md file but later opted to ansible guidelines and changed to this rst format for changelogs. Both Ansible and Terraform have different set of guidelines.